pkg/volume/flexvolume/flexvolume.go

```diff
@@ -103,7 +103,7 @@ func (plugin *flexVolumePlugin) getVolumeSource(spec *volume.Spec) *api.FlexVolu
 // NewMounter is the mounter routine to build the volume.
 func (plugin *flexVolumePlugin) NewMounter(spec *volume.Spec, pod *api.Pod, _ volume.VolumeOptions) (volume.Mounter, error) {
```

Comment:
Please update documentation and comments on `SecretRef` in `api/types.go` to reflect the new behavior.